### PR TITLE
post: js-ipfs 0.28

### PR DESCRIPTION
**Pull request by diasdavid**

### CODE REVIEW

**Comment by VictorBjelkholm** [comment]
@diasdavid maybe add the docker hub images as well?

File: content/post/33-js-ipfs-0.28.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+Now both IPFS and libp2p nodes expose the same PubSub API. This enables other applications that want to use the networking stack of IPFS only, including PubSub.
+
+Check the updated example in the js-libp2p repo at https://github.com/libp2p/js-libp2p/tree/master/examples/pubsub.
+
+### 💻 Multiple jsipfs CLI updates
+
+The jsipfs has been receiving a lot of love lately, updates include:
+  - `jsipfs ls -r`List directories recursively
+    - `jsipfs version` supports all flags now
```
Comment (VictorBjelkholm) [inline content/post/33-js-ipfs-0.28.md]: This part and for a few lines downwards seems to be indented strangly

**Comment by VictorBjelkholm** [comment]
@diasdavid let's merge this sooner rather than later. Is ready to be published?

**Comment by diasdavid** [comment]
@VictorBjelkholm I was just waiting on a review. Does it look good to you?

File: content/post/33-js-ipfs-0.28.md (hunk 1 above)
Comment (VictorBjelkholm) [reply content/post/33-js-ipfs-0.28.md]: @diasdavid only weird this ^

**Pull request merged**